src/client.ts

```diff
@@ -60,9 +59,38 @@ import {
 import {Transaction as LegacyTransaction} from "./legacy_transaction";
 import {Transaction, START_TRANSACTION_IMPL} from "./transaction";
 
-const PROTO_VER_MAJOR = 0;
-const PROTO_VER_MINOR_MIN = 9;
-const PROTO_VER_MINOR = 11;
+export class ProtocolVersion {
```

Comment:
IMO this is an overkill and should be done with two functions (`versionGreaterThan` and `versionGreaterThanOrEqual`) typed to accept a tuple of two numbers.

Reply:
what about using arrays of numbers? `[0, 11] >= [0, 10]` this _seems_ to work the same way comparing tuples in python works.

Reply:
changed

Reply:
> what about using arrays of numbers? `[0, 11] >= [0, 10]` this _seems_ to work the same way comparing tuples in python works.

I think this just converts them to strings, then compares them. So `[0, 11] >= [0, 9] === false`

Reply:
ok :+1:

Reply:
removed array comparison